src/meta/test/meta_bulk_load_service_test.cpp

```diff
@@ -283,6 +283,11 @@ class bulk_load_service_test : public meta_test_base
         return count;
     }
 
+    bool app_bulk_load_states_reset(int32_t app_id)
```

Comment:
the name of `app_bulk_load_states_reset`ave an ambiguity, I think `is_app_bulk_load_states_reset` is better